src/heretic/model.py

```diff
@@ -237,18 +391,62 @@ def abliterate(
                 else:
                     layer_refusal_direction = refusal_direction
 
-                # Projects any right-multiplied vector(s) onto the subspace
-                # spanned by the refusal direction.
-                projector = torch.outer(
-                    layer_refusal_direction,
-                    layer_refusal_direction,
-                ).to(self.model.dtype)
-
-                for matrix in matrices:
-                    # Ensure projector is on the same device as the matrix for multi-GPU support.
-                    device_projector = projector.to(matrix.device)
-                    # In-place subtraction is safe as we're not using Autograd.
-                    matrix.sub_(weight * (device_projector @ matrix))
+                for module in modules:
+                    # Skip modules without LoRA adapters (e.g., raw tensors in GPT-OSS)
+                    # PEFT only attaches LoRA to nn.Module instances, not raw tensors
```

Comment:
But we don't add any raw tensors in `get_layer_modules`, so this check shouldn't be necessary.